stdlib/public/core/Collection.swift

```diff
@@ -387,12 +387,11 @@ public protocol Collection: Sequence {
   /// Returns an iterator over the elements of the collection.
   override __consuming func makeIterator() -> Iterator
 
-  /// A sequence that represents a contiguous subrange of the collection's
-  /// elements.
+  /// A collection whose values represent contiguous subranges of this
```

Comment:
plural subranges?

Reply:
It's matching the plural values -- but I rewrote it to clarify.